data/conf/iceberg/llap/hive-site.xml

```diff
@@ -397,4 +397,39 @@
         <name>hive.lock.sleep.between.retries</name>
         <value>2</value>
     </property>
+    
+    <!-- -- Iceberg REST catalog specific properties -->
+    <property>
+        <name>iceberg.catalog.type</name>
+        <value>rest</value>
+    </property>
+    <property>
+        <name>iceberg.rest-catalog.uri</name>
+        <value>{restServerUrl}</value>
+    </property>
+    <!-- For OAUTH2 authentication add this as well -->
```

Comment:
@difin maybe drop the OAUTH2 configs here since half of them are missing + this PR doesn't support client authentication